test/db/cmd/cmd_di-on-signals

```diff
@@ -0,0 +1,549 @@
+EOF
+RUN
+
+
+NAME=di on SIGINT
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=2 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:0
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=suspended-by-user
+signal=none
+stopreason=suspended-by-user
+Hello, World!
+EOF
+RUN
+
+
+NAME=di on SIGIO
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=29 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:1
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=signal
+signal=none
+stopreason=signal
+Hello, World!
+EOF
+RUN
+
+
+NAME=di on SIGKILL
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=9 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:0
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=exit-pid
+signal=none
+stopreason=exit-pid
+EOF
+RUN
+
+
+NAME=di on SIGPIPE
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=13 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:1
+di~stopreason
+EOF
+EXPECT=<<EOF
+[+] signal 13 aka SIGPIPE received 0
+type=signal
+signal=SIGPIPE
+stopreason=signal
+Hello, World!
+EOF
+RUN
+
+
+NAME=di on SIGPROF
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=27 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:0
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=Profiling timer expired
+signal=SIGPROF
+stopreason=Profiling timer expired
+Hello, World!
+EOF
+RUN
+
+
+NAME=di on SIGQUIT
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=3 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:0
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=Quit
+signal=SIGQUIT
+stopreason=Quit
+Hello, World!
+EOF
+RUN
+
+
+NAME=di on SIGSEGV
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=11 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:0
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=Segmentation fault
+signal=SIGSEGV
+stopreason=Segmentation fault
+Hello, World!
+EOF
+RUN
+
+
+NAME=di on SIGSTKFLT
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=16 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:0
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=Stack fault
+signal=none
+stopreason=Stack fault
+Hello, World!
+EOF
+RUN
+
+
+NAME=di on SIGSTOP
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=19 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:0
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=Stopped
+signal=SIGSTOP
+stopreason=Stopped
+Hello, World!
+EOF
+RUN
+
+
+NAME=di on SIGSYS
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=31 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:0
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=Bad system call
+signal=SIGSYS
+stopreason=Bad system call
+Hello, World!
+EOF
+RUN
+
+
+NAME=di on SIGTERM
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=15 -d
+CMDS=<<EOF
+dc
+di~type
+di~signal:0
+di~stopreason
+EOF
+EXPECT=<<EOF
+type=Terminated
+signal=SIGTERM
+stopreason=Terminated
+Hello, World!
+EOF
+RUN
+
+NAME=di on SIGTRAP
+FILE=bins/elf/signals/signal-input
+ARGS=-e cfg.fortunes=false -e dbg.args=5 -d
```

Comment:
just add 1 test until you finish the code, or you will need to redo the boring tests part every time unnecessarily. also, the -e cfg.fortunes=false is not necessary